### DISCUSSION TITLE
it would be great to add the drawing tool similar to excalidraw

### DISCUSSION THREAD

**Discussion by faus1000** [jgraph/drawio#3058] [Ideas]
Hello, I have been seeing that a tool to make drawings (pencil - vectorial) such as excalidraw or paint would be very useful.

Is this possible through plugins? either
would it be a new feature?

**Comment by gal064** [comment]
This exists in the sketch theme. See here:
https://drawio-app.com/introducing-draw-ios-new-sketch-feature/
